### DISCUSSION TITLE
슬라이드 영역을 줄이고 위에 제목 영역을 추가하고 싶습니다

### DISCUSSION THREAD

**Discussion by mmcrs** [pronist/hELLO#202] [Q&A] [answered]
![ddddd](https://user-images.githubusercontent.com/85152618/220827934-6f6ae136-27d8-488f-83bf-4b6f37b54be9.png)

이전버전이었을 때는 꽉 차지 않는 슬라이드 영역이 됐었는데 최신인 4.1.2로 올리고나니 슬라이드가 전체화면밖에 지원하지 않는 것 같습니다. 그래서 원래대로 슬라이드 영역을 줄이고 위에 제목을 추가하고싶은데 어떻게 해야하나요?

**Comment by pronist** [comment, marked as answer]
@mmcrs 안녕하세요.

신 버전에는 3.x 에서 지원하던 좁은 슬라이드는 지원하지 않습니다. 이것을 처리하려면 스킨 커버를 추가하셔야 하는데, 단순 설정으로 끝날 문제는 아니므로 작성하셔야 할 코드가 제법 있을 것입니다. 원본 소스에서 스킨 커버를 정의하는 것은 다음과 같습니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/17f2d7d5e0e126d532087e36041585429f12272f/views/Main/Cover.pug#L67-L77

또한 **index.xml** 에 추가되어야 합니다.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/17f2d7d5e0e126d532087e36041585429f12272f/docs/index.xml#L32-L53
